src/dawn/IIR/ASTStringifier.h

```diff
@@ -0,0 +1,35 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
do we need multiple stringifiers? Wouldn't the one on the AST-Level be enough?

Reply:
There is only one AST stringifier. In this file there is only the alias.

Reply:
yes, but since there is no specialization there should not be aliasing. Just use the `ast::ASTStringifier`